
syntax = "proto3";

package helloworld;

service Greeter2 {
  rpc SayHello (HelloRequest) returns(HelloReply){}
}

message HelloRequest {
  string name = 1;
  int32 size_all = 2;
  repeated int32 size_arr = 3;
  int32 timeEpoch = 4;
  bytes datablob = 5;
}

message HelloReply {
  string name = 1;
  int32 size_all = 2;
  repeated int32 size_arr = 3;
  int32 timeEpoch = 4;
  bytes datablob = 5;
}

